packages/connection/src/Manager.php

```diff
@@ -1654,7 +1654,7 @@ public function public_xmlrpc_methods( $methods ) {
 	 * @param Array $args method call arguments.
 	 * @return an amended XMLRPC server options array.
 	 */
-	public function jetpack_getOptions( $args ) {
+	public function jetpack_get_options( $args ) {
```

Comment:
Are we OK changing this public function name?

Reply:
I'm okay with it specifically because it is relatively new.